Tests/BackendIntegrationTests/OtherIntegrationTests.swift

```diff
@@ -192,4 +192,33 @@ class OtherIntegrationTests: BaseBackendIntegrationTests {
         )
     }
 
+    func testRequestPaywallImages() async throws {
+        let offering = try await XCTAsyncUnwrap(try await self.purchases.offerings().current)
```

Comment:
I guess we are only checking the images for the current offering, which is probably ok 👍